src/db/util.py

```diff
@@ -104,3 +104,16 @@ def ensure_srid_definition(srid, proj4text):
 ON CONFLICT (srid) DO UPDATE SET proj4text = %s;'''
     params = (srid, proj4text, proj4text)
     run_statement(sql, params)
+
+
+def parse_connection_string(connection_string):
+    from urllib import parse
+    connection = parse.urlparse(connection_string, )
```

Comment:
Assert that connection.scheme == "postgresql" and that connection.netloc is not empty. Otherwise raise LaymanError